test/app/view/web/WebChessBoardView.spec.ts

```diff
@@ -133,3 +133,64 @@ describe('Web Chess Board View with starting pieces positions', () => {
     expect(a5Square).not.toHaveClass('square--possibleMove');
   });
 });
+
+describe('Web Chess Board View during move and capture methods', () => {
+  beforeEach(() => {
+    document.body.innerHTML = '';
+  });
+  const publishViewEventMock = jest.fn();
+  const viewEventBus: ViewEventBus = {
+    listenOn: jest.fn(),
+    publish: publishViewEventMock,
+  };
+  const chessBoardView: ChessBoardView = new WebChessView(viewEventBus);
+
+  it('move pawn from square A2 to A3', async () => {
+    const piecesPositions: PiecesBoardPositions = {
+      A2: { name: 'Pawn', side: Side.WHITE },
+    };
+    chessBoardView.showChessBoard(piecesPositions);
+    const a2Square = await screen.findByTestId('a2');
+    const a3Square = await screen.findByTestId('a3');
+    const a2WhitePawn = await screen.findByTestId('a2-img');
+
+    chessBoardView.movePiece('a2', 'a3');
```

Comment:
Dobrze testy napisane :). Tak mi się wydaje przynajmniej. Teraz trzeba dodać testy na presenter, że tam jakoś po kliknięciu najpierw w square na którym jest bierka, a potem na możliwy ruch to przesówa :)

Reply:
yup, póki co jestem na etapie ... łatwiej mówić, ciężej zrobić :-)